wiki/Medals/en.md

```diff
@@ -10,120 +10,16 @@
 
 All beatmaps in the pack have to be cleared in the [osu!](/wiki/Game_mode/osu!) game mode in order to qualify for the badge unless otherwise specified. The usage of either [difficulty reduction mods](/wiki/Game_modifier#difficulty-reduction) or [difficulty increase mods](/wiki/Game_modifier#difficulty-increase) is permitted.
```

Comment:
These paragraphs refer to the tables (i.e. "the list below", "unless otherwise specified", etc). They should either all be reworded, or have a link included below to their corresponding unlock requirements page.